lib/kvdb/kvdb_ctxn.c

```diff
@@ -697,7 +697,7 @@ kvdb_ctxn_set_create(struct kvdb_ctxn_set **handle_out, u64 txn_timeout_ms, u64
 
     memset(ktn, 0, sizeof(*ktn));
 
-    ktn->ktn_wq = alloc_workqueue("kvdb_ctxn_set", 0, 1);
+    ktn->ktn_wq = alloc_workqueue("hse_ctxn_reaper", 0, 1, 1);
```

Comment:
hse_kvdb_ctxn_reaper?

Reply:
Can't be any longer than 16 chars including the nul :-(

Reply:
Is that a self-imposed limit?

Reply:
Nevermind. https://man7.org/linux/man-pages/man3/pthread_setname_np.3.html

So strange...

What happens if I have two threads with the same name?

Reply:
We have that (see my thread list dump above) the kernel doesn't care, it's just an adornment for humans I suppose...
All threads in a workqueue get the same name, but sometimes we rename the long-lived ones to be more specific (like hse_kmc, hse_throttle, ...)